distributionviewer/core/static/js/app/components/views/chart.js

```diff
@@ -49,7 +49,20 @@ export class Chart extends React.Component {
     const chart = (
       <div className={`chart is-fetching chart-${this.props.chartId}`}>
         <Fetching />
-        <p className="chart-rollover-container"><span /><span /></p>
+        <table className="chart-rollover-table">
```

Comment:
We could've also gone with a `<dl>`. Feel free to ignore this a table is quite fine too.